Lib/test/test_warnings/__init__.py

```diff
@@ -877,6 +877,18 @@ def test_showwarning(self):
                                 file_object, expected_file_line)
         self.assertEqual(expect, file_object.getvalue())
 
+        # bpo-35178: Test custom formatwarning can receive line as positional
+        def formatwarning(message, category, filename, lineno, text):
+            return text
```

Comment:
I would prefer to use all arguments in the result, to check that all arguments are passed and in the correct order.

For example: return f'm={message}:c={category}:f={filename}:l={lineno}:t={text}'

Reply:
Changed it.